### PR TITLE
Add org.musesequencer.Muse3

### PR DESCRIPTION
**Pull request by hfiguiere**
This is a different software from MuseScore already on Flathub.

Jack seems to not be functional at the moment (I was told there was some recent regressions)

### CODE REVIEW

File: org.musesequencer.Muse3.json [hunk 1]
```diff
@@ -0,0 +1,201 @@
+{
+  "app-id": "org.musesequencer.Muse3",
+  "branch": "stable",
```
Comment (bilelmoussaoui) [inline org.musesequencer.Muse3.json]: Not needed

File: org.musesequencer.Muse3.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
```
Comment (bilelmoussaoui) [inline org.musesequencer.Muse3.json]: Tried 5.12?

File: org.musesequencer.Muse3.json [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        }
+      ]
+    },
+    {
+      "name": "muse3",
+      "buildsystem": "simple",
+      "build-commands": [
+        /* For some reason the cmake buildsystem in flatpak fail */
+        /* and we can't use the script in the tree as we need to set the prefix */
+        "mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=release -DCMAKE_INSTALL_PREFIX=/app ..",
```
Comment (bilelmoussaoui) [inline org.musesequencer.Muse3.json]: Can you paste the error you have here?

File: org.musesequencer.Muse3.json [hunk 4]
```diff
@@ -0,0 +1,201 @@
+{
+  "app-id": "org.musesequencer.Muse3",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "muse3",
+  "rename-desktop-file": "muse.desktop",
```
Comment (bilelmoussaoui) [inline org.musesequencer.Muse3.json]: It would be nice if you could upstream those changes.

**Comment by bilelmoussaoui** [comment]
testbot, build

**Comment by bilelmoussaoui** [comment]
testbot, build org.musesequencer.Muse3

**Comment by nedrichards** [comment]
bot, build org.musesequencer.Muse3

File: org.musesequencer.Muse3.json (hunk 2 above)
Comment (hfiguiere) [reply org.musesequencer.Muse3.json]: As I said on the other PR, jack doesn't build on 5.12

**Comment by hfiguiere** [comment]
re: failure, I'll remove the submodule.

**Comment by nedrichards** [comment]
bot, build org.musesequencer.Muse3
testbot, build org.musesequencer.Muse3

File: org.musesequencer.Muse3.json (hunk 2 above)
Comment (hfiguiere) [reply org.musesequencer.Muse3.json]: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/issues/608

Comment (barthalion) [reply org.musesequencer.Muse3.json]: As in the other PR, let's embed db and try switching to 5.12.

File: org.musesequencer.Muse3.json (hunk 4 above)
Comment (hfiguiere) [reply org.musesequencer.Muse3.json]: Done muse-sequencer/muse/#636

Comment (hfiguiere) [reply org.musesequencer.Muse3.json]: merged there.

File: org.musesequencer.Muse3.json (hunk 2 above)
Comment (hfiguiere) [reply org.musesequencer.Muse3.json]: Updated to 5.12 now

**Comment by barthalion** [comment]
bot, build org.musesequencer.Muse3
testbot, build org.musesequencer.Muse3

**Comment by barthalion** [comment]
Have you contacted upstream team if they would like to help with maintenance?

**Comment by hfiguiere** [comment]
I have upstreamed all the changes. Some of the patches come from them and will not be needed next time.

**Comment by nedrichards** [comment]
bot, build org.musesequencer.Muse3
testbot, build org.musesequencer.Muse3

**Comment by nedrichards** [comment]
@hfiguiere just to double check, is this blocked on JACK support?

**Comment by hfiguiere** [comment]
It works without Jack, using ALSA and RtAudio.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.musesequencer.Muse3

**Comment by nedrichards** [comment]
@hfiguiere let me know if there's any github usernames to add from upstream

**Pull request closed without merging**